frontend/src/bundles/common/components/button/styles.module.scss

```diff
@@ -0,0 +1,134 @@
+@import url("../../../../assets/css/variables.css");
+
+:root {
+    --font-family: var(--font-family);
+    --letter-spacing: var(--letter-spacing-button-text);
+    --gap: var(--px-2);
+    --border: 2px;
+    --border-radius: var(--px-2);
+    --transition-time: 0.3s;
+    --main-color: var(--gradient-6);
+    --hover-color: var(--gradient-5);
+    --disabled-color: var(--color-disable);
+    --medium-padding: var(--p-2) var(--p-4);
+    --medium-primary-padding-top-bottom: calc(var(--p-2) + var(--border));
+    --medium-primary-padding-right-left: calc(var(--p-4) + var(--border));
+    --small-padding: var(--p-1) var(--p-3);
+    --small-primary-padding-top-bottom: calc(var(--p-1) + var(--border));
+    --small-primary-padding-right-left: calc(var(--p-3) + var(--border));
+    --medium-font-size: var(--font-size-1-button-text);
+    --medium-font-weight: var(--font-weight-500);
+    --primary-color: var(--color-white-100);
+    --small-font-size: 16px;
+    --small-font-weight: var(--font-weight-400);
+    --secondary-text-color: #01cacc;
+    --secondary-hover-text-color: #0be9ee;
+    --plain-text-color: var(--color-pink-100);
+    --plain-hover-text-color: var(--color-blue-600);
+    --round-button-color: var(--color-pink-100);
+    --round-button-dimension: 28px;
+}
```

Comment:
Not sure why you need this

Reply:
Removed unnecessary variables in [a14e6a4](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/21/commits/a14e6a4d4d44118fcf3c291f456d982a528d7a7b)